src/pydata_sphinx_theme/assets/styles/components/_switcher-version.scss

```diff
@@ -31,23 +31,22 @@ button.btn.version-switcher__button {
     &:hover {
       background-color: var(--pst-color-surface);
     }
-  }
-
-  // Over-rides Bootstrap default blue for the current one (info-bg)
-  // TODO: @trallard to fix active in subsequent PR
-  a.list-group-item.active {
-    color: var(--pst-color-info);
-    position: relative;
-    z-index: 1;
+    &.active {
```

Comment:
Could we instead use the `current` `@mixin`? See https://github.com/pydata/pydata-sphinx-theme/blob/40c79bbdc2b8085b542ef0c6cd69420db2df0c8e/src/pydata_sphinx_theme/assets/styles/abstracts/_links.scss#L136-L137

This should account for any zoom-specific changes on the user side.

See use on the [`sidebar TOC`](https://github.com/pydata/pydata-sphinx-theme/blob/40c79bbdc2b8085b542ef0c6cd69420db2df0c8e/src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss#L205)

Reply:
OK, uses the mixin now. Thanks for pointing that out as an option (I didn't realize its `box-shadow:inset` would end up looking like `border-left`!)

Reply:
Yes, the box-shadow approach is quite nifty and helps avoid some of the bouncing effects on some components.